config/params/fieldparams/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@prysm//tools/go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "field_parameters_mainnet.go",
+        "field_parameters_minimal.go",  #keep
+    ],
```

Comment:
It turns out there is a better way to do this, which only includes the relevant file.

```
[ ] + select({
        "//config:mainnet": ["beacon_state.go"],
        "//config:minimal": ["beacon_state_minimal.go"],
    }),
```